js/util.js

```diff
@@ -0,0 +1,32 @@
+const getRandom = (min, max) => {
+  if (min>=max) {
+    return 0;}
+  else {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
```

Comment:
```suggestion
  if (min>=max) {
    return 0;
  } else {
    return Math.floor(Math.random() * (max - min + 1)) + min;
  }
```
синтаксис немного хромает. поправь везде пожалуйста